python/cudf/cudf/_lib/cpp/io/text.pxd

```diff
@@ -0,0 +1,27 @@
+# Copyright (c) 2020-2021, NVIDIA CORPORATION.
+
+from libcpp.memory cimport unique_ptr
+from libcpp.string cimport string
+
+from cudf._lib.cpp.column.column cimport column
+
+
+cdef extern from "cudf/io/text/data_chunk_source.hpp" \
+        namespace "cudf::io::text" nogil:
+
+    cdef cppclass data_chunk_source:
+        data_chunk_source() except+
+
+cdef extern from "cudf/io/text/data_chunk_source_factories.hpp" \
+        namespace "cudf::io::text" nogil:
+
+    unique_ptr[data_chunk_source] make_source(string data) except+
```

Comment:
```suggestion
    unique_ptr[data_chunk_source] make_source(string data) except +
```

Reply:
I don't mind to change it but a quick grep of the codebase shows 257 other places where `except+` has been used for years. If we want to always use `except +` for syntactical purposes we need to add a code formatting rule.

Reply:
@jdye64 sorry about this, I did the same `grep` but accidentally only checked `pyx` files and erroneously concluded that this wasn't done anywhere else. Since you've already made the changes we may as well stick with them, but you're right that this probably isn't an issue.